reference/security/index.md

```diff
@@ -7,13 +7,11 @@ applies_to:
 
 # Security
 
-This section of the documentation contains reference information for [{{elastic-sec}}](/solutions/security.md) features, including:
+This section contains reference information for {{elastic-sec}} features, including:
 
-* Prebuilt rules
-* Downloadable rule updates
-* Prebuilt jobs
-* Fields and object schemas
-* Endpoint command reference
-* Prebuilt anomaly detection jobs
+* [Fields and object schemas](/reference/security/fields-and-object-schemas/index.md)
+* [Endpoint command reference](/reference/security/endpoint-command-reference.md)
+* [Detection Rules Overview](//detection-rules/docs/index.md)
```

Comment:
This looks to be the correct page! 

Just made a minor tweak to the path, which should allow the cross-ref to resolve.

```suggestion
* [Detection Rules Overview](detection-rules://index.md)
```